config/routes.rb

```diff
@@ -10,7 +10,9 @@
 
   namespace :admin do
     resources :dashboard, only: [:index]
-    resources :companies, only: [:show, :destroy, :update]
+    resources :companies, only: [:show, :destroy, :update] do
+      resources :notes, only: [:index, :create, :destroy]
+    end
     resources :pdfs, only: [:create, :new]
     resource :tools_resources, only: [:edit]
   end
```

Comment:
Do we really want routes for notes? I would think that this should just be on the company edit page.